nexus/src/external_api/console_api.rs

```diff
@@ -85,7 +89,9 @@ pub async fn login_spoof(
         let authn_opctx = nexus.opctx_external_authn();
         let session = nexus.session_create(&authn_opctx, user_id).await?;
 
-        let mut response = http_response_see_other(String::from("/"))?;
+        let mut response = HttpResponseHeaders::new_unnamed(HttpResponseOk(
+            SpoofLoginResponse {},
+        ));
```

Comment:
This was kind of a weird API, had to look at the Dropshot source to find a way that works. But I get the feeling what I'm trying to do here is a little unusual.